src/shipper/shipper.config.ts

```diff
@@ -28,14 +28,26 @@ export class LogShipper {
    */
   onLog: LogProcessFunction[] = [];
 
-  static async load(): Promise<LogShipper> {
+  static async load(logger?: typeof Log): Promise<LogShipper> {
     if (LogShipper.INSTANCE == null || LogShipper.INSTANCE.isRefreshNeeded) {
       const configName = process.env[Env.ConfigName] ?? DefaultParameterStoreBasePath;
       const data = await SsmCache.get(configName);
       const cfg = LogShipperConfigValidator.parse(data);
 
-      const accounts = await Promise.all(
-        cfg.map((c) => SsmCache.get(c).then((account) => LogShipperConfigAccountValidator.parse(account))),
+      const accounts: LogShipperConfigAccount[] = [];
```

Comment:
This enables multiple accounts to be configured from a single SSM value.

(Im sure I had this in master at some stage, not sure how i lost it)